src/Suave.LibUv/Tcp.fs

```diff
@@ -298,7 +308,7 @@ type LibUvServer(maxConcurrentOps, bufferManager, logger : Logger,
     Marshal.FreeCoTaskMem handle
 
   member this.closeHandler (handle : IntPtr) (arg : IntPtr) =
-    uv_close(handle,uv_close_cb(this.closeHandlerCallback))
+    uv_close(handle,this.uv_close_cb_handler)
 
   member this.uv_stop_loop (_ : IntPtr) =
```

Comment:
Should we name F# methods in this manner? I understand why p-invoke method should; but F# ones?